### ISSUE TITLE
Can't delete record in search

### ISSUE THREAD

**Issue by SnijdeA** [MiniKeePass/MiniKeePass#351]
When I open a database on my iPhone and do a search, I can't delete records from the found ones.
There is no delete option in edit and swipe doesn't work.
How do I delete records while in a search?

**Comment by rundlesm** [comment]
Yeah its a bit annoying as the search results don't indicate which folder the entry has been found.

I don't know the answer but one thing I noticed is that the search result includes entries in Recycle Bin.
So, I guess deleting from Search may be prevented for that reason.

How I do it is to use the desktop client to search for and delete entries, then I resync the kdbx file with Dropbox on the iPhone.

HTH

Steve
